MAUI/NumericEntry/UpDown-Button.md

```diff
@@ -91,6 +91,48 @@ sfNumericEntry.UpDownPlacementMode = NumericEntryUpDownPlacementMode.InlineVerti
 
 ![UpDown Vertical Placement in .NET MAUI NumericEntry](UpDownButton_images/UpDownButtonPlacementVertical.png)
 
+## UpDown button alignment
+
+You can customize the `NumericEntry` control button alignment by using the `UpDownButtonAlignment` property. Set it to **Left** to position both buttons on the left, or **Both** to place them at the extreme ends.  
```

Comment:
You can adjust the alignment of the UpDown buttons in the NumericEntry control using the UpDownButtonAlignment property. Set its value to Left, Right, or Both to position the buttons on the left, right, or both sides of the entry field, respectively.

Reply:
Updated